refactored_minitwit_tests.py

```diff
@@ -10,12 +10,13 @@
     :license: BSD, see LICENSE for more details.
 """
 import requests
+import os
 
 
 # import schema
 # import data
 # otherwise use the database that you got previously
-BASE_URL = "http://0.0.0.0:4567"
+BASE_URL = os.getenv('MINITWIT_BASE_URL', 'http://127.0.0.1:4567')
```

Comment:
Hvorfor er det nødvendigt at have `os.getenv('MINITWIT_BASE_URL'`?

Reply:
Når man kører testen i docker compose skal den bruge `http://minitwit:5000` som base url, men det skal være noget andet når man den lokalt (localhost) - så jeg tænkte det var smartest at kunne overskrive den pr. miljø

Reply:
Er det her noget vi gør på cli hver gang, eller bør vi lave en `.env.example` fil så folk kan lave en lokal .env fil til det?

Reply:
Den defaulter til localhost, så man behøver ikke sætte den. Men det kan godt være vi skal lave noget eksempel env på et tidspunkt om ikke andet bare for dokumentation.